frontend/feat/feature-flags.json

```diff
@@ -15,18 +15,18 @@
       },
       "description": "Toggle the new header, including recent searches."
```

Comment:
Can we remove the `new_header` flag, too?

Reply:
Done in ff12982.